netmanager/src/views/pages/Download/Download.js

```diff
@@ -211,15 +211,6 @@ const Download = (props) => {
     await downloadDataApi(fileType.value, data, fileType.value === "csv" , outputFormat.value)
       .then((response) => response.data)
       .then((resData) => {
-        if(isEmpty(resData)) {
```

Comment:
Thanks for this @Codebmk ....

- Just wondering why this is not causing the same challenge in production environment. Is this error handling not present in the master branch? If it is also present in the master branch, then why isn't the problem replicated there? Any thoughts on this?
- Also, why is the data empty? What it causing this? What is the response from the endpoints? Are we logging the resData variable? What is the output for devices/Sites which we know are online and sending data? 
cc: @FaithDaka @ja-odur

Reply:
I have found something interesting. It works when you choose `AQCSV` and fails on `AirQo Standard`. Lemme re-verify.

Reply:
Yeap. Verified. I used the same values for those two formats but it worked for `AQCSV` and failed on `AirQo Standard`. So the trouble is not with the error handler.

Reply:
@Baalmart for UrbanBetter the data is actually not there. I have tested in postman